[2024-08-17T00:32:54.007+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-17T00:32:54.018+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: predict_values_TS_11.get_cases_dataset.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-17T00:32:54.022+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: predict_values_TS_11.get_cases_dataset.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-17T00:32:54.023+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-17T00:32:54.032+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_cases_dataset.get_datasets_from_google> on 2024-08-04 00:00:00+00:00
[2024-08-17T00:32:54.039+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=22347) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-17T00:32:54.041+0000] {standard_task_runner.py:63} INFO - Started process 22388 to run task
[2024-08-17T00:32:54.040+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'predict_values_TS_11', 'get_cases_dataset.get_datasets_from_google', 'scheduled__2024-08-04T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmprtmvevf1']
[2024-08-17T00:32:54.042+0000] {standard_task_runner.py:91} INFO - Job 39: Subtask get_cases_dataset.get_datasets_from_google
[2024-08-17T00:32:54.077+0000] {task_command.py:426} INFO - Running <TaskInstance: predict_values_TS_11.get_cases_dataset.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [running]> on host 08898c2f83c5
[2024-08-17T00:32:54.132+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ruslan Solarev' AIRFLOW_CTX_DAG_ID='predict_values_TS_11' AIRFLOW_CTX_TASK_ID='get_cases_dataset.get_datasets_from_google' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-04T00:00:00+00:00'
[2024-08-17T00:32:54.133+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-17T00:33:13.985+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-17T00:33:13.986+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-17T00:33:13.992+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=predict_values_TS_11, task_id=get_cases_dataset.get_datasets_from_google, run_id=scheduled__2024-08-04T00:00:00+00:00, execution_date=20240804T000000, start_date=20240817T003254, end_date=20240817T003313
[2024-08-17T00:33:14.049+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-17T00:33:14.065+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-17T00:33:14.127+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
